python/ray/experimental/dask/scheduler.py

```diff
@@ -196,7 +196,8 @@ def _rayify_task(task, key, deps):
         # Ray properly tracks the object dependencies between Ray tasks.
         object_refs, repack = unpack_object_refs(args, deps)
         # Submit the task using a wrapper function.
-        return dask_task_wrapper.remote(func, repack, *object_refs)
+        return dask_task_wrapper.options(name=str(key)).remote(
```

Comment:
This will yield a process title of e.g. `ray::(array-abcde12345, 0, 0, 1)` and a PID of `(array-abcde12345, 0, 0, 1)` in the dashboard. I'm wondering if adding a `dask::` prefix to the task name might be prudent, which would yield `ray::dask::(array-abcde12345, 0, 0, 1)` and `dask::(array-abcde12345, 0, 0, 1)`, respectively.

Reply:
Opting for a `dask:` prefix on the task name, since the current dashboard frontend does a naive string split on `"::"`, assuming that there is only one such separator in the string.